Whiskey/en-US/about_Whiskey_Writing_Tasks.help.txt

```diff
@@ -121,16 +121,14 @@ LONG DESCRIPTION
 
     The `NodePath` string is the name of the key where the path to Node should be saved in the `TaskParameter` hashtable when calling your task.
 
-    To control what version of Node to use, users must specify an `engines.node` property in their package.json file, e.g.
+    If the user doesn't use the version parameter to control what version of Node to use, Whiskey also looks in the user's package.json file at the `engines.node` property, e.g.
 

```

Comment:
Is it worth noting that if you include a version in your task definition or properties, that any information in `engines.node` of your package.json will be ignored?

Reply:
Yes, but I want to get this merged in ASAP.